packages/babel-plugin-transform-react-jsx-source/test/fixtures/react-source/with-source/exec.js

```diff
@@ -0,0 +1,8 @@
+var actual = transform(
+  'var x = <sometag __source="custom" />;',
+  Object.assign({}, opts, { filename: '/fake/path/mock.js' })
```

Comment:
I believe it's a global exposed to test fixtures from [here](https://github.com/babel/babel/blob/master/packages/babel-helper-transform-fixture-test-runner/src/index.js#L106).